src/satipc.c

```diff
@@ -421,12 +422,12 @@ int satipc_open_device(adapter *ad)
 		if (sip->rtcp < 0)
 			LOG("Could not listen on port %d: err %d: %s", sip->listen_rtp + 1, errno, strerror(errno));
 
-		ad->fe_sock = sockets_add(ad->fe, NULL, ad->id, TYPE_TCP,
+		ad->fe_sock = sockets_add(ad->fe, &sv, ad->id, TYPE_TCP,
 								  (socket_action)satipc_reply, (socket_action)satipc_close,
 								  (socket_action)satipc_timeout);
 		sip->rtcp_sock = -1;
 		if (sip->rtcp >= 0)
-			sip->rtcp_sock = sockets_add(sip->rtcp, NULL, ad->id, TYPE_TCP,
+			sip->rtcp_sock = sockets_add(sip->rtcp, &sv, ad->id, TYPE_TCP,
```

Comment:
You cannot add the destination of a socket to another socket

Reply:
Why not? Without this change the ADDRESS of the remote SAT>IP server is not printed in the log. With this simple change the address is printed, that's more easy to understand.

Reply:
I understand you are adding it because of logging, but basically you are binding multiple sockets to the same destination, which makes no sense and could lead to hidden bugs.